Orange/tests/test_dasktable.py

```diff
@@ -111,3 +111,14 @@ def test_domain_basic_stats(self):
             self.assertEqual(c.max, 50)
             self.assertAlmostEqual(c.mean, 22.5328063)
             self.assertAlmostEqual(c.var, 84.4195562)
+
+    def test_sample(self):
+        zoo = Table('zoo')
+        with named_file('', suffix='.hdf5') as fn:
+            DaskTable.save(zoo, fn)
+            dzoo = DaskTable.from_file(fn)
+            dzoo_sample = dzoo.sample(10)
```

Comment:
Please try to be specific. In this case `dzoo.sample(size=10)`, but according to another comment up there, rename `size`. :)